### PR TITLE
Implement NodeGetVolumeStats RPC and bugfixes

### PR DESCRIPTION
**Pull request by Praveenrajmani**
- Add xfs_quota support to set the hard limits for a volume
- Fix the formatting to cover only xfs

### CODE REVIEW

**Comment by Praveenrajmani** [comment]
Tested the feature. The PR is ready for the review.

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ FROM alpine:latest
 
 WORKDIR /
 RUN apk add xfsprogs
+RUN apk add xfsprogs-extra
```
Comment (wlan0) [inline Dockerfile:16]: This should not be needed?

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,7 @@ FROM alpine:latest
...
 RUN apk add e2fsprogs
```
Comment (wlan0) [inline Dockerfile:17]: This can be removed

File: Dockerfile [hunk 3]
```diff
@@ -13,6 +13,7 @@ FROM alpine:latest
...
 RUN apk add dosfstools
```
Comment (wlan0) [inline Dockerfile:18]: This can be removed

File: Dockerfile (hunk 1 above)
Comment (wlan0) [reply Dockerfile:16]: Nvm, this is this for the quota right?

File: pkg/node/publish_unpublish.go [hunk 4]
```diff
@@ -48,7 +48,7 @@ func PublishVolume(ctx context.Context, stagingPath, containerPath string, readO
 	}
 
 	if shouldBindMount {
-		opts := []string{"bind"}
+		opts := []string{"bind", "prjquota"}
```
Comment (wlan0) [inline pkg/node/publish_unpublish.go:51]: What happens if there is no storage requirement specified on the requested volume?

**Review by wlan0 — COMMENTED** [review]
Minor comments.

File: Dockerfile (hunk 1 above)
Comment (Praveenrajmani) [reply Dockerfile:16]: yes

File: pkg/node/publish_unpublish.go (hunk 4 above)
Comment (Praveenrajmani) [reply pkg/node/publish_unpublish.go:51]: As we run xfs_quota on these mounts, We will add this option whenever we do a bind mount.

**Review by wlan0 — APPROVED** [review]

**Pull request merged**